lib/stm32/common/crypto_common_f24.c

```diff
@@ -36,6 +36,10 @@
 
 #define CRYP_CR_ALGOMODE_MASK	((1 << 19) | CRYP_CR_ALGOMODE)
 
+#ifndef __REV
+#define __REV(x) __builtin_bswap32(x)
+#endif
```

Comment:
out of scope of this commit! please make this separate, if it's even necessary. (it shuldn't be)

Reply:
This is needed for the the crypto to work correctly, I had tried to grep the libopencm3 and found the exiting defines at 
```
./include/libopencmsis/core_cm3.h:62:#define __REV(x) __builtin_bswap32(x)
```
But see nowhere use this headers. what is the best place to put this define?